josm-presets/de-signals-eso.xml

```diff
@@ -1528,11 +1515,6 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						display_values="Formsignal (Sh 0/Sh 1),Lichtsignal (Hp 0/Sh 1),Zugdeckungssignal (Hp 0/Kennlicht)"
```

Comment:
The values line here is wrong, the last ';' should be ','. Not your fault, but if you fix the marker light stuff, then please fix that, too.